bin/pynumenc

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+"""Convert an input value to/from bytes/numbers."""
+
+import argparse
+import sys
+
+import numenc
+import pynumenc_meta
+
+
+def main() -> None:
+    """
+    Execute the main routine.
+    """
+    parser = argparse.ArgumentParser(description=pynumenc_meta.__description__)
+    parser.add_argument("conversion_type",
+                        help="The direction and input type of the conversion.\n"
+                             "For instance, to_int16 or from_float64")
+    parser.add_argument(
+        "value",
+        help="The value to be converted. For bytes, in hexadecimal. "
+             "For instance, deadbeef or 1992.")
+
+    supported_types = pynumenc_meta.__supported_types__
```

Comment:
I would put only package-related information into pynumenc_meta. Consider moving supported_types to this script or to `numenc/__init__.py`.